packages/patternfly-4/react-charts/src/components/ChartUtils/chart-theme.ts

```diff
@@ -128,76 +198,6 @@ export const getDonutUtilizationTheme = (themeColor: string, themeVariant: strin
   return theme;
 };
 
-// Returns dark theme colors
-export const getDarkThemeColors = (themeColor: string) => {
-  switch (themeColor) {
-    case ChartThemeColor.blue:
-      return DarkBlueColorTheme;
-    case ChartThemeColor.cyan:
-      return DarkCyanColorTheme;
-    case ChartThemeColor.gold:
-      return DarkGoldColorTheme;
-    case ChartThemeColor.gray:
-      return DarkGrayColorTheme;
-    case ChartThemeColor.green:
-      return DarkGreenColorTheme;
-    case ChartThemeColor.multi:
-    case ChartThemeColor.multiOrdered:
-      return DarkMultiColorOrderedTheme;
-    case ChartThemeColor.multiUnordered:
-      return DarkMultiColorUnorderedTheme;
-    case ChartThemeColor.orange:
-      return DarkOrangeColorTheme;
-    case ChartThemeColor.purple:
-      return DarkPurpleColorTheme;
-    default:
-      return DarkBlueColorTheme;
-  }
-};
-
-// Returns light theme colors
-export const getLightThemeColors = (themeColor: string) => {
-  switch (themeColor) {
-    case ChartThemeColor.blue:
-      return LightBlueColorTheme;
-    case ChartThemeColor.cyan:
-      return LightCyanColorTheme;
-    case ChartThemeColor.gold:
-      return LightGoldColorTheme;
-    case ChartThemeColor.gray:
-      return LightGrayColorTheme;
-    case ChartThemeColor.green:
-      return LightGreenColorTheme;
-    case ChartThemeColor.multi:
-    case ChartThemeColor.multiOrdered:
-      return LightMultiColorOrderedTheme;
-    case ChartThemeColor.multiUnordered:
-      return LightMultiColorUnorderedTheme;
-    case ChartThemeColor.orange:
-      return LightOrangeColorTheme;
-    case ChartThemeColor.purple:
-      return LightPurpleColorTheme;
-    default:
-      return LightBlueColorTheme;
-  }
-};
-
-// Applies theme color and variant to base theme
-export const getTheme = (themeColor: string, themeVariant: string): ChartThemeDefinition => {
-  // Deep clone
-  const baseTheme = {
-    ...JSON.parse(JSON.stringify(ChartBaseTheme))
-  };
-  switch (themeVariant) {
-    case ChartThemeVariant.dark:
-      return merge(baseTheme, getDarkThemeColors(themeColor));
-    case ChartThemeVariant.light:
-      return merge(baseTheme, getLightThemeColors(themeColor));
-    default:
-      return merge(baseTheme, getLightThemeColors(themeColor));
-  }
-};
-
```

Comment:
Why was moving these functions necessary? They were in alphabetical order

Reply:
Reverted function order